.gitignore

```diff
@@ -29,3 +29,7 @@ rhino.iws
 
 .java-version
 .kotlin
+
+# android specific
+android-sdk/
+local.properties
```

Comment:
required for the `install-android-sdk.sh` script. Should we provide such script or rely that everyone can install the SDK on their machines?